SecretSanta/test/SecretSanta.Api.Tests/Controllers/GroupControllerTests.cs

```diff
@@ -32,13 +33,13 @@ public void GetAllGroups_ReturnsGroups()
 
             var service = new Mock<IGroupService>();
             service.Setup(x => x.FetchAll())
-                .Returns(new List<Group> { group1, group2 })
+                .Returns(Task.FromResult(new List<Group> { group1, group2 }))
```

Comment:
There is a convenience extension method `ReturnsAsync`
```suggestion
                .ReturnsAsync(new List<Group> { group1, group2 })
```